apps/customer/views.py

```diff
@@ -81,4 +82,124 @@ def create(self, request, *args, **kwargs):
         return create_instance(self, request, *args, **kwargs)
 
     def update(self, request, *args, **kwargs):
-        return update_instance(self, request, *args, **kwargs)
\ No newline at end of file
+        return update_instance(self, request, *args, **kwargs)
+    
+#==========================================================================
+
+class CustomerCreateViews(generics.CreateAPIView,mi.ListModelMixin,mi.CreateModelMixin,mi.RetrieveModelMixin,mi.UpdateModelMixin,mi.DestroyModelMixin):
+    queryset = Customer.objects.all()
+    serializer_class = CustomerSerializer
+    
+    def get(self, request, *args, **kwargs):
+        if 'pk' in kwargs:
+            return self.retrieve(request, *args, **kwargs)
+        return list_all_objects(self, request, *args, **kwargs)
```

Comment:
Do not use multiple return statements here, use single return statement.

Reply:
Updated